### DISCUSSION TITLE
Toggle Mechanical Chime

### DISCUSSION THREAD

**Discussion by xAcrosonicx** [tsightler/ring-mqtt#748] [Support Q&A] [answered]
Apologies if this has been brought up before, I am struggling to find any information regarding the topic.

I have a mechanical chime and see there is an option through my doorbell cam to toggle “ring in-home chime”. Does Ring MQTT allow us access to toggle this feature? I would like to disable it when my infant is in her crib and re-enable it when she leaves.

thanks in advance!

**Comment by tsightler** [comment, marked as answer]
Pretty much everything that is supported is already listed in the documentation, if it's not listed there, then it probably isn't supported thus no reason to struggle to find anything.  This feature is not supported and there are no current plans to add this feature, feel free to submit a PR and I would consider it.
